AutoscoperM/AutoscoperMLib/IO.py

```diff
@@ -265,7 +235,7 @@ def writeTRA(fileName: str, transforms: list[vtk.vtkMatrix4x4]) -> None:
             traFile.write(",".join(row) + "\n")
 
 
-def writeTemporyFile(filename: str, data: vtk.vtkImageData) -> str:
+def writeTemporaryFile(filename: str, data: vtk.vtkImageData) -> str:
```

Comment:
I suggest to fix the typos in `writeTemporyFile` and `removeTemporyFile` in its own pull request (either before or after this one)

Reply:
> these functions are actually no longer being used.

To reduce technical debt, I removed these two functions.

> hmmm.. what is used in the #99 ? When registration are output?

It seems that when the "Export Transforms" button is clicked in the 3DHierarchicalRegistration module, the function [`AutoscoperM/AutoscoperMLib/IO.py:writeTRA(fileName, transforms)`](https://github.com/sbelsk/SlicerAutoscoperM/blob/remove-VRG-generation/AutoscoperM/AutoscoperMLib/IO.py#L229) is used (not affected by this PR).